.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
i think you can do the pre commit config in package.json like we did in PT and events helsinki projects. but if that works, no worries.

Reply:
It did not work because in the monorepo the package root and git root are in different folders, so this had to be done.

Reply:
ok